listings/views.py

```diff
@@ -7,8 +10,9 @@
 
 class ListingViewSet(viewsets.ModelViewSet):
     """ViewSet for the Listing model."""
-    queryset = Listing.objects.all()
+    queryset = Listing.objects.all().prefetch_related('images')
     serializer_class = ListingSerializer
+    permission_classes = [IsAuthenticated]
 
     def list(self, request, **kwargs):
```

Comment:
same for this guy

Reply:
Ye

Reply:
I guess I can take them out for now. They'll get removed eventually anyway once we write more complicated views.